docs/docs/user/run-node/03-txs.md

```
@@ -179,6 +179,42 @@ func sendTx() error {
+func sendTx() error {
+    // --snip--
+	txBuilder.SetUnordered(true)
+	txBuilder.SetTimeoutTimestamp(time.Now())
+}
+```
+
+Unordered transactions from the same account must use a unique timeout timestamp value. The difference between each timeout timestamp value may be as small as a nanosecond, however.
+
+```go
+import (
+	"github.com/cosmos/cosmos-sdk/client"
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)